packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -148,23 +151,29 @@ function ConnectorDetails() {
               </div>
             </div>
             <div className={styles.operations}>
-              <Button
-                title="connector_details.check_readme"
-                size="large"
-                onClick={() => {
-                  setIsReadMeOpen(true);
-                }}
-              />
-              <Drawer
-                title="connectors.title"
-                subtitle="connectors.subtitle"
-                isOpen={isReadMeOpen}
-                onClose={() => {
-                  setIsReadMeOpen(false);
-                }}
-              >
-                <Markdown className={styles.readme}>{data.readme}</Markdown>
-              </Drawer>
+              {isEmailServiceConnector ? (
```

Comment:
No need for this condition? Seems that checking `data.usage` is enough

Reply:
At the moment, the usage feature is only defined for the `EmailServiceConnector`.
We have not yet defined the usage-related features for other connectors, such as Email or SMS. 
So, I think it would be more explicit and clear to use `isEmailServiceConnector` in this case.

Reply:
Same wondering, do we have usage data for other connectors? How about social connectors?

Reply:
No, only for `EmailServiceConnector`.

Reply:
Updated, when the `usage` is not `undefined` and the connector is an email connector, the usage info will be displayed.